Specifications/QIR/Data-Types.md

```diff
@@ -133,7 +105,7 @@ allocated by the caller because the length of the string depends on the actual v
 | __quantum__rt__range_to_string   | `%String*(%Range)`   | Returns a string representation of the range. |
 | __quantum__rt__bigint_to_string  | `%String*(%BigInt*)` | Returns a string representation of the big integer. |
 
-### Big Integers
+## Big Integers
 
```

Comment:
The spec is not primarily meant to reflect what we implement, but is meant as a shared convention for different implementers. What is and isn't implemented should hence be documented as part of our compiler and runtime rather than on the spec here.